tgui/packages/tgui/styles/interfaces/BankingComputer.scss

```diff
@@ -0,0 +1,33 @@
+/**
+ * @file
+ * @copyright 2024
+ * @author mloccy (https://github.com/mloccy)
+ * @license MIT
+ */
+
+@use '../colors.scss';
+@use '../base.scss';
+
+.headerRow {
+  font-size: larger;
+}
+
+.budgetButtonCell {
+  padding-top: 5px;
+  float: right;
+}
+
+.budgetStatusCell {
+  padding-top: 5px;
+  padding-bottom: 5px;
+  vertical-align: middle;
+}
```

Comment:
Not sure, but I feel like this can be probably be done with props instead of style overrides?